src/System.Management.Automation/utils/tracing/TracingGen.cs

```diff
@@ -37,7 +37,6 @@ public sealed partial class Tracer : System.Management.Automation.Tracing.EtwAct
         /// </summary>
         public const long KeywordAll = 0xFFFFFFFF;
 
-        private static readonly Guid providerId = Guid.Parse("a0c1853b-5c40-4b15-8766-3cf1c58f985a");
```

Comment:
Is there a reason why this doesn't use the `PowerShellCore` Provider ID `f90714a8-5509-434a-bf6d-b1624c8a19a2`?

Reply:
It is already there. It is inherited from EwtActivity.ProviderId (that is PSEtwLogProvider.ProviderGuid).